config/sync/config_split.config_split.develop.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 3b60f64a-7bb3-42b3-ae27-3af99742acba
+langcode: en
+status: true
+dependencies: {  }
+id: develop
+label: Develop
+description: 'Configuration just for the develop branch / dev environment.'
+weight: 0
+stackable: false
+no_patching: false
+storage: folder
+folder: ../config/develop
+module:
```

Comment:
SAMLAuth and External Auth should be enabled in all environments from prod down to local developer's laptops.  The signing keys and some config will vary, but everyone will use some kind of SAML auth to authenticate in all environments.

Reply:
This works, though adding samlauth to `core.extensions.yml` would be less confusing.